spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -203,8 +216,7 @@ public Subscriber createSubscriber(String subscriptionName, MessageReceiver rece
 	}
 
 	@Override
-	public PullRequest createPullRequest(String subscriptionName, Integer maxMessages,
-			Boolean returnImmediately) {
+	public PullRequest createPullRequest(String subscriptionName, Integer maxMessages, Boolean returnImmediately) {
```

Comment:
This looks like automatic reformatting in your IDE.
I personally prefer not more than `120` symbols, but guys might have different vision, e.g. `90`.
That's may be why we had such a code before.

Reply:
Fixed - changed to 90 characters or less.